src/content/get-started/overview.mdx

```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
 
 In order to fully install Okteto, you'll need the following:
 
-- A domain to which you can add a wildcard DNS record
 - A Kubernetes cluster
+- An Okteto License
+- A domain to which you can add a wildcard DNS record
 - A working installation of kubectl
 - A working installation of Helm v3 (v3.8 or higher)
-- An Okteto License
-
-### Domain
-
-You'll need sufficient access to a [subdomain](reference/helm-chart-values.mdx#subdomain) to add a wildcard DNS record, such as dev.example.com.
-By default, all endpoints created by Okteto for your development environments will be exposed on the wildcard subdomain you choose.
 
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow your cloud provider's Kubernetes cluster creation guide.
-Okteto supports Kubernetes versions 1.24 through 1.26.
+Okteto supports Kubernetes versions 1.25 through 1.28.
 
 We recommend the following specs:
-- v1.26
+- v1.28
 - A pool with at least 3 nodes with a minimum of 4CPUs and 16 GB of Memory
 - 250 GB per disk
 
 You'll be using the cluster's API server endpoint when configuring Okteto.
 
 > Our installation guides assume Okteto will be running in a dedicated cluster. We recommend [contacting our team](https://www.okteto.com/contact/) if you plan on installing Okteto in a cluster with other workloads.
 
+### License
+
+You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
```

Comment:
You also receive a license on a trial. We should reword this to prevent confusion.  We have wording in other pages on this.

Reply:
@rberrelleza could you share the right wording here?

Reply:
I think what we have on https://deploy-preview-596--okteto-docs.netlify.app/docs/1.17/self-hosted/manage/okteto-license/#how-to-obtain-a-license is a good fit for this block.